client/state/selectors/get-site-timezone-string.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Return the site setting `timezone_string` value from the state-tree
+ *
+ * @param  {Object}  state - Global state tree
+ * @param  {Number}  siteId - Site ID
+ * @return {?String} site setting timezone
+ */
+export default function getSiteTimezoneString( state, siteId ) {
```

Comment:
> Do we standardize this on the backend?

We are doing _something strange_ :-p only into the PostSchedule component. We've been talking about this for a long time. In the following screenshot we are showing the Site timezone location and also the offset.

<img src="https://cloud.githubusercontent.com/assets/77539/22439829/e06bc492-e70f-11e6-9e35-aa032f0628b1.png" width="500px" />

The goal is trying to let them aware to the users what will be the post date when under this settings conditions.

Let me update the branch before to add more comments.